packages/cadl-rlc-test/test/openai/cadl-output/src/parameters.ts

```diff
@@ -1,17 +1,50 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { RawHttpHeadersInput } from "@azure/core-rest-pipeline";
 import { RequestParameters } from "@azure-rest/core-client";
-import { EmbeddingsRequest, CompletionsRequest } from "./models";
+
+export interface EmbeddingsHeaders {
+  /** The ID of the end-user, for use in tracking and rate-limiting. */
+  user?: string;
+}
 
 export interface EmbeddingsBodyParam {
-  body?: EmbeddingsRequest;
+  body?: {
+    input_type: string;
```

Comment:
@joheredi We implement the CADL spread alias as anonymous model, because the alias info would be erased so there is no model name for this body type.

Do you have any concern for this design?

https://github.com/Azure/autorest.typescript/blob/b6734113ecaef100844d0231c89fa453cd142145/packages/cadl-rlc-test/test/openai/spec/models/completions.create.cadl#L9-L26

Reply:
Yeah, if we can't get a model name from CADL I prefer to have the anonymous model definition. This will protect us from generating awkward model names